sqlite-web/config.json

```diff
@@ -5,36 +5,17 @@
   "description": "Explore your SQLite database",
   "url": "https://github.com/hassio-addons/addon-sqlite-web/tree/main/README.md",
   "ingress": true,
```

Comment:
`"ingress": true` but ingress port is no longer specified?

Reply:
Nope, still uses Ingress, but now using its default port 8099